### ISSUE TITLE
site level permission changes from public to private not detected

### ISSUE THREAD

**Issue by ChalithaUdara** [maoo/alfresco-indexer#12]
Hi Maurizio,

When we change the permissions of **initially public site to a private** one, Permission changes are not properly detailed in node details webscript. The issue is that in alfresco even the public site is changed to private, it allows **base.ReadPermissions** to **GROUP_EVERYONE**. but **site.SiteConsumer** which is originally GROUP_EVERYONE changes accordingly. This could be Alfresco issue. 

```
 "readableAuthorities" : [
"GROUP_EVERYONE"
,
"GROUP_site_newsite2_SiteManager"
,
"GROUP_site_newsite2_SiteCollaborator"
,
"GROUP_site_newsite2_SiteContributor"
,
"GROUP_site_newsite2_SiteConsumer"

],
```
should change to

```
"readableAuthorities" : [
"GROUP_site_newsite2_SiteManager"
,
"GROUP_site_newsite2_SiteCollaborator"
,
"GROUP_site_newsite2_SiteContributor"
,
"GROUP_site_newsite2_SiteConsumer"

],
```

Thanks,
Chalitha

**Comment by maoo** [comment]
Sorry for the delay on this @ChalithaUdara ; is this PR still requested/needed? I can merge it and include it in the next release (0.8.1)

**Comment by ChalithaUdara** [comment]
Yes. it would be good to have this included. I can again do a test run to cover this use case once it is merged

**Comment by maoo** [comment]
Ok, I've merged locally, run tests and all is good; however, before merging, I'd advise to cover the new logic with tests; @ChalithaUdara , is this something you could provide?

**Comment by ChalithaUdara** [comment]
We have tested the logic and it worked as expected (this logic is covered by our product test cases at a higher level). So I think it is safe to merge this. I will provide test-cases to cover new logic for indexer webscripts project, but not immediately. Will do as soon as I can find some free time
